packages/react-components/react-persona/src/components/Persona/usePersonaStyles.ts

```diff
@@ -21,16 +21,18 @@ const avatarSpacing = `--fui-Persona__avatar--spacing`;
 const useStyles = makeStyles({
   base: {
     display: 'inline-grid',
-    gridAutoColumns: 'max-content',
     gridAutoRows: 'max-content',
   },
+
   after: {
     gridAutoFlow: 'column',
     justifyItems: 'start',
+    gridTemplateColumns: 'max-content [middle] auto',
```

Comment:
In order for the wrap to work, the column containing the text needs to be set to auto. If it's set to max-content it will ignore the container's size.